packages/blade/src/components/Box/LayoutPrimitivesDocs.tsx

```diff
@@ -44,7 +44,7 @@ function LayoutPrimitivesDocs(): JSX.Element {
       componentName="Layout Primitives"
```

Comment:
can we add some text in the boxes? the empty boxes in dark color makes it very difficult to understand the intent we are conveying for someone who doesn't has a lot of context about box and layout
![image](https://user-images.githubusercontent.com/11384858/225219374-355d56cf-0518-4d88-a1fe-0d2480ff0c53.png)

Reply:
done. Added `Box1` and `Box2`

Reply:
can we also add this to responsive props section? seems that is missed. actually just check wherever it is required lets add it

Reply:
done

Reply:
padding 🙈 
![image](https://user-images.githubusercontent.com/11384858/225236330-b5df5742-a1f4-450e-aea7-45a831e3e27e.png)

Reply:
done